lib/external_interaction/rubix/rubix_external_rpc.dart

```diff
@@ -39,4 +40,19 @@ class RubixExternalService extends RubixExternalServiceBase {
         ..receiver = event.receiver;
     });
   }
+
+  @override
+  Future<OrgStatus> approveOrgAuthRequest(
+      ServiceCall call, OrgAuthRequest request) async {
+    var user = RubixService.getAuthUser(call);
+    const rubix = 'rubix';
```

Comment:
this must have been coming from phone,

Reply:
no, no, not need as this is rubix ext rpc. But could have simply passed this to the next function without having to store it in a variable.

Reply:
Ok will make that change.